# Request

---
#Response
RTCTime wake_time
